src/main/java/org/terasology/equipment/system/CharacterScreenClientSystem.java

```diff
@@ -29,13 +29,23 @@
 import org.terasology.rendering.nui.NUIManager;
 import org.terasology.utilities.Assets;
 
+/**
+ * Client system that handles how equipment-related information is displayed.
+ */
 @RegisterSystem(RegisterMode.CLIENT)
 public class CharacterScreenClientSystem extends BaseComponentSystem {
     @In
     private NUIManager nuiManager;
 
     private static final String CHARACTER_SCREEN_NAME = "Equipment:BackupScreen";
 
+    /**
+     * This method is executed when the player presses the 'character screen' button
```

Comment:
Nitpick: Good to end the first sentence with a `.` (javadoc parser looks for that)